tvtk/vtk_parser.py

```diff
@@ -640,7 +643,10 @@ def _find_get_set_methods(self, klass, methods):
 
         # Find the default and range of the values.
         if gsm:
+            # Useful for debugging on failures:
```

Comment:
For future evolution of this code, how about using logging in place of the commented-out print statements?

Reply:
This code is primarily used at build time, so would logging be appropriate?  I was thinking of using an environment variable like "MAYAVI_DEBUG_BUILD" or so to turn on these print statements.  What do you recommend?

Reply:
Not sure, but just from a code cleanliness point of view I'd really like to get away from having commented-out code in the codebase.

A "verbose" or "debug" flag to turn on the print statements doesn't seem unreasonable; that's probably better than using an environment variable at this point - we probably want the environment variable handling in the topmost layer.